<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_Window</class>
 <widget class="QMainWindow" name="Settings_Window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.485, y1:0.415045, x2:0.480294, y2:0.932, stop:0 rgba(61, 255, 232, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="minimize_button">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="asset_loader.qrc">
              <normaloff>:/icons/assets/minimize_button.svg</normaloff>:/icons/assets/minimize_button.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exit_button">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="asset_loader.qrc">
              <normaloff>:/icons/assets/x.svg</normaloff>:/icons/assets/x.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QLineEdit" name="video_save_path"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="video_save_location_label">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="asset_loader.qrc">:/icons/assets/video_save_location_text.svg</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="audio_save_location_label">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="asset_loader.qrc">:/icons/assets/audio_save_location_text.svg</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="video_save_button">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="asset_loader.qrc">
              <normaloff>:/icons/assets/save_text.svg</normaloff>:/icons/assets/save_text.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="audio_save_path"/>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="audio_save_button">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="asset_loader.qrc">
              <normaloff>:/icons/assets/save_text.svg</normaloff>:/icons/assets/save_text.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="vidlocbrowser_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="asset_loader.qrc">
              <normaloff>:/icons/assets/folder.svg</normaloff>:/icons/assets/folder.svg</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="audlocbrowser_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="asset_loader.qrc">
              <normaloff>:/icons/assets/folder.svg</normaloff>:/icons/assets/folder.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="asset_loader.qrc"/>
 </resources>
 <connections/>
</ui>
